lib/config.js

```diff
@@ -110,6 +110,19 @@ class Config {
     return opc;
   }
 
+  /**
+   * Parse a set of options for Streaming Process
+   *
+   * @param {Object} params
+   * @return an object with streaming process parameters.
+   */
+  parseStreamProcessParams(params = {}) {
```

Comment:
This is not really necessary just for one option.

Reply:
Ok, I close de PR. Thanks